syz-fuzzer/fuzzer.go

```diff
@@ -602,8 +661,41 @@ func triageInput(pid int, env *ipc.Env, inp Input) {
 	}
 }
 
-func execute(pid int, env *ipc.Env, p *prog.Prog, needCover, minimized, candidate bool, stat *uint64) []ipc.CallInfo {
+func executeHintSeed(pid int, env *ipc.Env, p *prog.Prog) {
+	if !kcovCompsEnabled {
+		return
+	}
+	// first execute the original program to dump comparisons from KCOV
```

Comment:
Full line comments start from a capital letter and end with dot throughout the codebase. Please follow this style.
It's fine to use non-capital letter and no dot for comments that are on the same line with code, e.g.:
```
     foo() // just foo
```